deployment/deployer.yaml

```diff
@@ -67,6 +67,16 @@ items:
       - watch
       - delete
       - update
+  - apiVersion: v1
```

Comment:
Again,  adding or removing labels on notes is a very highly privileged operation. It seems a little bit odd to make the nodes adjust to fit a daemonset.  Does this mean we're not setting nodes up correctly?